HW5/Homework_5.py

```diff
@@ -0,0 +1,17 @@
+import time
+
+# 1
+variable = input('Input your variable!\n')
+for i in variable:
+    if i.isalnum():
+        if i.isdigit():
+            print(f"It's {['odd', 'even'][int(i) % 2 == 0]} number!")
+        else:
+            print(f"It's {['lower', 'upper'][i.isupper()]} letter!")
+    else:
+        print("It's a symbol!")
+
+# 2
+while x := 1 > 0:
```

Comment:
To create an infinite loop, you can simply use `while True` since you don't need the `x` variable here